mcp/health-tool-mcp/server.py

```
@@ -860,60 +861,68 @@ def fhirTemplateGeneration(
+                return _log_output_and_return(
+                    tool_name="fhirTemplateGeneration",
+                    request_id=_request_id,
+                    caller=_caller,
+                    response_str=format_error_output(
+                        "fhirTemplateGeneration",
+                        "Empty directory",
+                        f"Empty definitions at: {fhir_spec_directory}. {setup_hint}",
+                    ),
+                    start_time=_start_time,
+                )
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)